docfiles/docs.html

```diff
@@ -11,8 +11,11 @@
 	<script type="text/javascript">
 		var _jipt = [];
 		_jipt.push(['project', '@crowdinproject@']);
-		</script>
-		<script type="text/javascript" src="//cdn.crowdin.com/jipt/jipt.js"></script>
+		_jipt.push(['escape', function () {
+			window.location.href = window.location.href.replace(/[$\?]translate=1/, '');
```

Comment:
will this handle "?foo=1&translate=1"  ?

Reply:
yes this will handle it